physionet-django/console/views.py

```diff
@@ -243,8 +240,7 @@ def submission_info_redirect(request, project_slug):
     return redirect('submission_info', project_slug=project_slug)
 
 
-@login_required
-@user_passes_test(is_admin, redirect_field_name='project_home')
+@permission_required('physionet.change_section', raise_exception=True)
```

Comment:
```suggestion
@permission_required('project.change_publishedproject', raise_exception=True)
```

The permission names are confusing - PublishedProject permissions manage all sorts of projects, i.e. not published ones (ActiveProject) etc. The permissions should be renamed to e.g. `project.xxx_project`

Reply:
@kshalot will take care of this